packages/atlas-experiment-table/src/head/SelectionTableHeaderCell.js

```diff
@@ -27,7 +27,7 @@ class SelectionTableHeaderCell extends React.Component {
     super(props)
 
     this.state = {
-      fileTypes: props.downloadFileTypes.map(fileType => fileType.id)
+      fileTypes: props.downloadFileTypes && props.downloadFileTypes.map(fileType => fileType.id)
```

Comment:
By giving a default prop value you can remove the conditional check altogether, and your logic remains clean and straightforward. The fewer edge cases, the better.